src/platform/windows.rs

```diff
@@ -102,6 +102,9 @@ pub trait WindowExtWindows {
 
   /// Returns the current window theme.
   fn theme(&self) -> Theme;
+
+  /// Not to display window icon in the task bar.
```

Comment:
Maybe replace with; Whether to display window icon in the taskbar.

Reply:
yeah that sounds better, I will change the linux one too since that's where I copied it from xD

Reply:
Actually, I think a better comment would be `Removes the window icon from the task bar` since `skip_taskbar` is a one way function (I think we can make it both way but it is better be in another PR ).